lib/permissions/permission-schema.json

```diff
@@ -1,26 +1,97 @@
 {
     "study": {
-        "PI": [
+        "pi": [
             "read_Study",
             "update_Study",
             "read_Scan"
         ],
-        "Coordinator": [
+        "coordinator": [
             "read_Study"
         ],
-        "DataEntry": [
+        "dataentry": [
             "read_Study"
         ],
-        "Scanner": [
+        "scanner": [
```

Comment:
Scanner seems like a site role, since it will be sending scans for multiple studies...